source/fundamentals/indexes.txt

```diff
@@ -397,6 +397,34 @@ Refer to the :manual:`Unique Indexes page </core/index-unique>` in the MongoDB s
 
 .. driver-content-end
 
+Clustered Indexes
+~~~~~~~~~~~~~~~~~
+
+**Clustered indexes** are indexes that improve the performance of
+insert, update, and delete operations on **clustered collections**.
+Clustered collections store documents ordered by the clustered index key
+value.
+
+To create a clustered index, specify the ``ClusteredIndexOptions`` in
+the ``CreateCollectionOptions``. The ``ClusteredIndexOptions`` must
+specify the ``_id`` field as the key, and the unique field as ``true``.
+
+The following example creates a clustered index on the ``_id`` field in
+the ``tea.vendors`` collection:
+
+.. code-block:: java
+   
+   MongoDatabase database = mongoClient.getDatabase("tea");
+   ClusteredIndexOptions cioptions = new ClusteredIndexOptions(Filters.eq("_id", 1), true);
+   CreateCollectionOptions coptions = new CreateCollectionOptions().clusteredIndexOptions(cioptions);
```

Comment:
Issue: the variable names should follow variable naming convention. Generally, this means that you should use words instead of first letters of words, and camelCase between the words.

Suggestion:
ClusteredIndexOptinos clusteredIndexOptions = ...
CreateCollectionOptions createCollectionOptions = ...